src/Service/PictureService.php

```diff
@@ -32,36 +35,22 @@ public function storePicture(string $filePath, string $originalFileName, string
 
         $this->dm->persist($file);
 
-        [$description, $embeddings] = $this->generateDescriptionAndEmbeddings($filePath);
+        $image = new Imagine()->open($filePath);
 
         // Create a new Face document
         $face = new Face();
         $face->name = $name;
         $face->file = $file;
-        $face->resizedImage = $this->getResizedImage($filePath);
-        $face->description = $description;
-        $face->embeddings = $embeddings;
+        $face->resizedImage = $this->getResizedImage($image);
+        $face->embeddings = $this->voyageAI->generateEmbeddings($image->get(Format::ID_PNG));
+        $face->description = '';
```

Comment:
Leaving this field in for the time being, but we may end up not generating a description